### DISCUSSION TITLE
set focus to same element when formviewer is loaded

### DISCUSSION THREAD

**Discussion by ManaliV** [optimajet/formengine#46] [Q&A]
Can I set focus to some form element when form viewer is loaded? We are using below to create form viewer, We are using custom component.
```
<FormViewer
      view={DefaultBuilderView()}
      getForm={() =>formTemplate}
      viewerRef={ref}
      initialData={formState}
      onFormDataChange={handleFormDataChange} />
```

**Comment by sergeythrees** [comment]
Hi, @ManaliV 
you can add `autoFocus` attribute through `HTML-attrbutes` editor:
![chrome_JkycjKBloi](https://github.com/user-attachments/assets/4bd02800-5680-4a6c-9ea5-049c848e6660)
